lang/sk.yml

```diff
@@ -3,9 +3,9 @@ sk:
     Add: 'Pridať {name}'
     AddMultipleOptions: Pridať
   GridFieldSiteTreeState:
-    Draft: '<i class="btn-icon gridfield-icon btn-icon-pencil"></i> Uložený ako koncept: {date}'
+    Draft: '<i class="font-icon-pencil btn--icon-md"></i> Uložený ako koncept: {date}'
     Modified: zmenené
-    Published: '<i class="btn-icon gridfield-icon btn-icon-accept"></i> Publikovaný: {date}'
```

Comment:
It's a shame that this HTML is in the translations - ideally we would remove it now - would you mind doing that @hdpero?

```diff
- return _t(
-     __CLASS__ . '.Draft',
-     '<i class="font-icon-pencil btn--icon-md"></i> Saved as Draft on {date}',
+ return '<i class="font-icon-pencil btn--icon-md">' . _t(
+     __CLASS__ . '.Draft',
+     'Saved as Draft on {date}',
```

Reply:
Yea, I would be happy to change it.